functions/supplier/file-supplier/src/test/java/org/springframework/cloud/fn/supplier/file/DefaultFileSupplierTests.java

```diff
@@ -16,26 +16,28 @@
 
 package org.springframework.cloud.fn.supplier.file;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Hmm. I think you may not have the spring-cloud formatting on this project in intelliJ? (one of us doesn't as mine tries to put it back where it was).